exp-player/addon/components/exp-player.js

```diff
@@ -142,6 +143,7 @@ able to continue the study.
         next() {
             console.log('next');
             var frameIndex = this.get('frameIndex');
+            this.sendAction('updateFrameIndex', frameIndex);
```

Comment:
Yes, that same `frameIndex` is passed into the exp-player via the template [here](https://github.com/CenterForOpenScience/isp/pull/64/files#diff-2f922b8396c42d2f74d61c16c2624b7fR10). When the value is updated on the controller it will also get updated on the component.